functions.php

```diff
@@ -216,6 +216,10 @@ function wpspf_get_form_field_ajax(){
 			<div class="col">Field default value:</div>
 			<div class="col"><input type="text" name="wpspf_input_field_default_value" class="field" id="wpspf_input_field_default_value"></div>
 		</div>';
+		$fieldHtml .='<div class="row">
```

Comment:
Added MaxLength attribute since description (and other fields) have a character limit according to the authorize.net docs.